.github/workflows/ci.yml

```diff
@@ -20,8 +20,11 @@ env:
 
 jobs:
   lint:
+    needs: build-setup
```

Comment:
As we saw before the windows test job should also build with the sentry rustflags, if the secret is available (as all other unit tests)